s2jobs/src/main/scala/org/apache/s2graph/s2jobs/task/Sink.scala

```diff
@@ -182,14 +200,40 @@ class ESSink(queryName:String, conf:TaskConf) extends Sink(queryName, conf) {
 
 /**
   * S2graphSink
+  *
   * @param queryName
   * @param conf
   */
-class S2graphSink(queryName:String, conf:TaskConf) extends Sink(queryName, conf) {
+class S2graphSink(queryName: String, conf: TaskConf) extends Sink(queryName, conf) {
   override def mandatoryOptions: Set[String] = Set()
+
   override val FORMAT: String = "org.apache.s2graph.spark.sql.streaming.S2SinkProvider"
 
-  override protected def writeBatch(writer: DataFrameWriter[Row]): Unit =
-    throw new RuntimeException(s"unsupported source type for ${this.getClass.getSimpleName} : ${conf.name}")
+  private def bulkload(df: DataFrame): Unit = {
+    val options = S2GraphHelper.toGraphFileOptions(conf)
```

Comment:
*toGraphFileOptions* method could handle only GraphFileOptions's parameters.
If we pass other parameters it will throw exception follows:
```
failed to parse options...
java.lang.IllegalArgumentException: failed to parse options...
	at org.apache.s2graph.s2jobs.loader.GraphFileOptions$.toOption(GraphFileOptions.scala:82)
```